src/registrar/admin.py

```diff
@@ -1694,13 +1694,13 @@ def queryset(self, request, queryset):
         "last_submitted_date",
         "last_status_update",
         "status",
-        "generic_org_type",
-        "federal_type",
-        "federal_agency",
-        "organization_name",
         "custom_election_board",
-        "city",
-        "state_territory",
+        "converted_generic_org_type",
```

Comment:
It would be good to add  @admin.display definitions (see other areas in admin.py that do this) so that the headers of these columns do not say "converted" in the title.

Reply:
Thank you for pointing that out!

Reply:
I still don't see updates for this.  Did we decide against it?

Reply:
I added them under filters.